.travis.yml

```diff
@@ -16,6 +16,7 @@ script: "make test xref dialyzer"
 branches:
   only:
     - master
+    - presjimbench
```

Comment:
That's not needed, I'll update the travis branches when I implement the perftest!